lib/twilio/call.ts

```diff
@@ -1296,6 +1296,22 @@ namespace Call {
    */
   declare function muteEvent(isMuted: boolean, call: Call): void;
 
+  /**
+   * Emitted when the {@link Call} has regained media connectivity.
+   * @param call - The {@link Call}.
+   * @example `call.on('reconnected', () => { })`
+   * @event
+   */
+  declare function reconnected(): void;
```

Comment:
```suggestion
  declare function reconnectedEvent(call: Call): void;
```
Does this event need to be post-fixed with `-event`?